helm/hpcc/templates/eclccserver.yaml

```diff
@@ -80,6 +81,7 @@ data:
             accessDali: "yes"
             accessEsp: "yes"
         spec:
+          {{- include "hpcc.placementMatchName" (dict "root" $ "pod" "compile-" "group" "eclccserver") | indent 10 }}
```

Comment:
similar to prev. comment - would be good to include .name, so multiple targets could have different placements.